### PR TITLE
Kanderson/us112834 attach rubrics

### PR DESCRIPTION
**Pull request by kieranderson**
Moving rubrics under the evaluation editor, and moving to own rubrics folder

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-secondary.js [hunk 1]
```diff
@@ -78,14 +77,10 @@ class AssignmentEditorSecondary extends SaveStatusMixin(RtlMixin(EntityMixinLit(
 				.token="${this.token}">
 			</d2l-activity-assignment-editor-submission-and-completion>
 
-			<d2l-activity-rubrics-list-container
```
Comment (pureooze) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-secondary.js]: 😄 yay

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-evaluation-editor.js [hunk 2]
```diff
@@ -114,6 +125,7 @@ class ActivityAssignmentEvaluationEditor extends LocalizeMixin(LitElement) {
 					${this._renderAnnotationsSummary()}
 					${this._renderTurnitinSummary()}
 				</ul>
+				${this._renderRubricsCollectionEditor()}
```
Comment (pureooze) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-evaluation-editor.js]: Will this also need to show up in the summary?

File: components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubric-editor.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { css, html, LitElement } from 'lit-element/lit-element.js';
 import { Association } from 'siren-sdk/src/activities/Association.js';
 import { EntityMixinLit } from 'siren-sdk/src/mixin/entity-mixin-lit.js';
 import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
-import { SaveStatusMixin } from './save-status-mixin';
+import { SaveStatusMixin } from '../save-status-mixin';
 
 class ActivityRubricsEditor extends SaveStatusMixin(RtlMixin(EntityMixinLit((LitElement)))) {
```
Comment (pureooze) [inline components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubric-editor.js]: Does this still need the `SaveStatusMixin`?

Comment (kieranderson) [reply components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubric-editor.js]: For now yes.

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-evaluation-editor.js (hunk 2 above)
Comment (kieranderson) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-evaluation-editor.js]: I believe so, in a future PR

File: components/d2l-activity-editor/d2l-activity-rubrics/lang/en.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+/* eslint quotes: 0 */
+
+export default {
+	"hdrRubrics": "Rubrics" //Header for the rubrics section
```
Comment (ppaskaris-d2l) [inline components/d2l-activity-editor/d2l-activity-rubrics/lang/en.js]: do you have to do anything for this to get picked up by whatever localizes the strings?

Comment (ppaskaris-d2l) [reply components/d2l-activity-editor/d2l-activity-rubrics/lang/en.js]: I think you have to add this file to this thing: https://github.com/BrightspaceHypermediaComponents/activities/blob/master/activities.serge.json

**Pull request merged**